pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1847,78 +1830,71 @@ func (cc *XController) updateQueueJobStatus(queueJobFromAgent *arbv1.AppWrapper)
 }
 
 func (cc *XController) worker() {
-	defer func() {
-		if pErr := recover(); pErr != nil {
-			klog.Errorf("[worker] Panic occurred error: %v, stacktrace: %s", pErr, string(debug.Stack()))
-		}
-	}()
+	//	defer func() {
```

Comment:
This was a catcher to cause MCAD panic issues, until we have extensive tests ready, I think we should add this back

Reply:
Sure, I will add it in the final commit. I just wanted to trap some errors.